bundles/admin/admin-layereditor/components/TextAreaInput.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Input from 'antd/lib/input';
+
+export const TextAreaInput = (props) => {
+    const { TextArea } = Input;
```

Comment:
We could move this out of the component scope so that it's done only once; not every render invocation.